### PR TITLE
Create "misc_commands" enum

### PR DESCRIPTION
**Pull request by wolfreak99**
I have replaced the sections that used magic numbers with a "misc_command" enum, which can be used for commands that aren't really gamestate specific. this will help with banishing magic numbers and also banish the need to rely on looking commands up by a string variable, which is vulnerable to typos etc

### CODE REVIEW

File: src/openrct2/game.h [hunk 1]
```diff
@@ -96,6 +96,12 @@ enum GAME_COMMAND {
 	GAME_COMMAND_COUNT
 };
 
+enum MISC_COMMAND {
+	MISC_COMMAND_CHAT = -1,
+	MISC_COMMAND_TOGGLE_SCENERY_CLUSTER = -2,
+	MISC_COMMAND_PASSWORDLESS_LOGIN = -3,
+};
```
Comment (janisozaur) [inline src/openrct2/game.h]: Is there any point in having these available in network-less builds?

These should rather be declared in guarded part in `network.h`.

Comment (wolfreak99) [inline src/openrct2/game.h]: You make a point... I'll fix that once I get home.

File: src/openrct2/network/network.cpp [hunk 2]
```diff
@@ -1570,7 +1570,7 @@ void Network::Server_Handle_AUTH(NetworkConnection& connection, NetworkPacket& p
 		bool passwordless = false;
 		if (connection.AuthStatus == NETWORK_AUTH_VERIFIED) {
 			const NetworkGroup * group = GetGroupByID(GetGroupIDByHash(connection.Key.PublicKeyHash()));
-			size_t actionIndex = NetworkActions::FindCommandByPermissionName("PERMISSION_PASSWORDLESS_LOGIN");
+			size_t actionIndex = NetworkActions::FindCommand(MISC_COMMAND_PASSWORDLESS_LOGIN);
```
Comment (Gymnasiast) [inline src/openrct2/network/network.cpp]: It looks like you can replace these two lines with
```
passwordless = group->CanPerformAction(MISC_COMMAND_PASSWORDLESS_LOGIN);
```

Comment (wolfreak99) [inline src/openrct2/network/network.cpp]: @Gymnasiast wouldn't it make sense to use group->CanPerformCommand?

**Pull request merged**